packages/teleterm/src/services/tshd/types.ts

```diff
@@ -14,34 +14,54 @@ import apiApp from './v1/app_pb';
 import apiService from './v1/service_pb';
 import apiAuthSettings from './v1/auth_settings_pb';
 import apiAccessRequest from './v1/access_request_pb';
+import apiUsageEvents from './v1/usage_events_pb';
 
 export type Application = apiApp.App.AsObject;
+
 export interface Kube extends apiKube.Kube.AsObject {
   uri: uri.KubeUri;
 }
+
 export interface Server extends apiServer.Server.AsObject {
   uri: uri.ServerUri;
 }
+
 export interface Gateway extends apigateway.Gateway.AsObject {
   uri: uri.GatewayUri;
   targetUri: uri.DatabaseUri;
 }
+
 export type AccessRequest = apiAccessRequest.AccessRequest.AsObject;
 export type ResourceId = apiAccessRequest.ResourceID.AsObject;
 export type AccessRequestReview = apiAccessRequest.AccessRequestReview.AsObject;
+
 export interface GetServersResponse
   extends apiService.GetServersResponse.AsObject {
   agentsList: Server[];
 }
+
 export interface GetDatabasesResponse
   extends apiService.GetDatabasesResponse.AsObject {
   agentsList: Database[];
 }
+
 export interface GetKubesResponse extends apiService.GetKubesResponse.AsObject {
   agentsList: Kube[];
 }
+
 export type GetRequestableRolesResponse =
   apiService.GetRequestableRolesResponse.AsObject;
+
+export type UsageEventWithDate = Modify<
```

Comment:
Can we rename it to `ReportUsageEventRequest`? The existence of this separate type made sense on my branch with those weird duplicated oneofs, but here if we just want to enrich an existing type, I think we should follow how it's done for other types and just declare a new one with the same name but changed fields.

Reply:
Done